mypyc/primitives/str_ops.py

```diff
@@ -144,11 +144,25 @@
     arg_types=[str_rprimitive],
     return_type=bit_rprimitive,
     c_function_name='CPyStr_IsTrue',
-    error_kind=ERR_NEVER,
-)
+    error_kind=ERR_NEVER)
 
 str_ssize_t_size_op = custom_op(
     arg_types=[str_rprimitive],
     return_type=c_pyssize_t_rprimitive,
     c_function_name='CPyStr_Size_size_t',
     error_kind=ERR_NEG_INT)
+
+# obj.decode()
+# obj.decode(encoding)
+# obj.decode(encoding, errors)
+decode_types: List[RType] = [bytes_rprimitive, str_rprimitive, str_rprimitive]
+decode_constants: List[Tuple[int, RType]] = [(0, pointer_rprimitive),
+                                             (0, pointer_rprimitive)]
+for i in range(len(decode_types)):
```

Comment:
The for loop doesn't save a lot of code and makes this harder to understand. I'd prefer just having three normal primitive definitions.